### PR TITLE
[13.0][ADD] hr_attendance_user_list: New addon.

### PR DESCRIPTION
**Pull request by victoralmau**
This module adds the list of employee attendance (read only).

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT34897

### CODE REVIEW

File: hr_attendance_user_list/views/hr_attendance_view.xml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                        context="{'group_by': 'check_out'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="hr_attendance_user_list_action" model="ir.actions.act_window">
+        <field name="name">My Attendances</field>
+        <field name="res_model">hr.attendance</field>
+        <field name="view_mode">tree</field>
```
Comment (chienandalu) [inline hr_attendance_user_list/views/hr_attendance_view.xml]: Maybe a pivot would be handy here?

**Review by chienandalu — APPROVED** [review]
Tested :+1:

**Review by CarlosRoca13 — APPROVED** [review]
Tested

**Pull request merged**